test.js

```diff
@@ -16,6 +16,13 @@ describe('markdown-it-implicit-figures', function() {
     assert.equal(res, expected);
   });
 
+  it('should add <figure> when image is by itself in a paragraph or preceeded by a standalone link', function () {
+    md = Md().use(implicitFigures, { dataType: true, figcaption: true });
+    var src = 'text with ![](img.png)\n\n![](fig.png)\n\nanother paragraph\n\n[![Caption](fig.png)](http://example.com)\n\nyet another paragraph';
```

Comment:
Hm, I did not remove the [figcaption](https://github.com/arve0/markdown-it-implicit-figures/blob/master/test.js#L21)?